CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ set(SSG_MINOR_VERSION 1)
 set(SSG_PATCH_VERSION 31)
 set(SSG_VERSION "${SSG_MAJOR_VERSION}.${SSG_MINOR_VERSION}.${SSG_PATCH_VERSION}")
 
-set(SSG_VENDOR "ssgproject" CACHE STRING "Specify the XCCDF 1.2 vendor string.")
+set(SSG_VENDOR "open-scap" CACHE STRING "Specify the XCCDF 1.2 vendor string.")
```

Comment:
org should be part of the vendor string IMO, it's not part of the XCCDF ID pattern, without it being in the pattern we are unable to say that com.redhat is the vendor. at best we can say that org.redhat is the vendor.